terminologies/technique/electrocorticogram.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+ "@id":"https://openminds.ebrains.eu/controlledTerms/technique/electrocorticogram",

```

Comment:
this one should be deleted. This is "electrocorticography"

Reply:
Didn't exist anymore in PanProject. Should be ok after rerunning script.